wordpress-mobile/WordPress-Android
# Feature/glide media

Fixes #7963 

Replaces Volley with Glide in Media -> adds support for animated gif images.

LocalImageTask was used to show images before they were uploaded to the server. I've removed it, since Glide supports displaying local images out of the box.

To test:
Affects 4 screens - Only difference from the user perspective should be, that gif image are animated.

- Media Grid Adapter: `My Site -> Media`
- Media Settings: reachable either from `My Site -> Media -> select an item` or `Editor -> add image -> click on the image`
- Media Preview: `My Site -> Media -> select an item -> click on the image in the header` or `My Site -> Media -> Add new Image -> select it before it's fully uploaded`
- Stock Photos: `Editor -> add new Media -> click on the image icon -> Choose from Free Photo Library`

Test - regular images, gif images and videos. Displaying local images (before they are fully uploaded) might be also worth testing.
